### PR TITLE
index: 20201026 Blockchain Learning Challenge

### PR DESCRIPTION
**Pull request by xaur**
Review: https://github.com/decredcommunity/events/blob/challenge/index/20201026.1.yml

### CODE REVIEW

File: index/20201026.1.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+title: Blockchain Learning Challenge - Decred
+lang: es
+title_es: { any? remove if not needed }
```
Comment (adcadde) [inline index/20201026.1.yml]: Same as EN, remove.

File: index/20201026.1.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    announcements:
+      - https://twitter.com/Decred_ES/status/1327347110455480321
+    presenters: elian
+    media:
+      - https://www.youtube.com/watch?v=kqbejo0sA5c
+  - title: Decred Learning Challenge Q&A Party
+    start_utc: 2020-11-13 22:00
+    announcements:
+      - https://twitter.com/Decred_ES/status/1327357791967846401
+    presenters: { who? }
```
Comment (adcadde) [inline index/20201026.1.yml]: elian, adcade

File: index/20201026.1.yml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    media:
+      - { link? }
```
Comment (adcadde) [inline index/20201026.1.yml]: Was right after 'How to become a Decred contractor?' and we stopped the recording, so no media link for this one.

File: index/20201026.1.yml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  - pablito
+attendance:
+  - 136 people registered for the Learning Phase from Argentina, Bolivia, Chile, Colombia, Cuba, México, Nicaragua, Paraguay, Perú, Uruguay, Venezuela
+  - 15 teams organized for the competition
+  - 6 teams presented projects in the final round
+description: The goal of the challenge is to share tools and practical abilities about blockchain technology in order for participants to develop an original project using Decred public blockchain data. First phase is one week of workshops about Bitcoin and blockchain, alternative consensus protocols, decentralised organisations and governance, dcrdata API and blockchain data, and Q&A party. Second phase is original project development with mentoring and support. At the end participants must release their projects with ISC license and present them in 3-minute video pitches.
+media:
+  - url: https://twitter.com/Decred_ES/status/1325817130978340868
+    note: Decred Talent Land Blockchain Challenge begins Nov-9
+  - url: https://www.youtube.com/watch?v=kBjgLcDZW8k
```
Comment (adcadde) [inline index/20201026.1.yml:74]: https://youtu.be/kBjgLcDZW8k

File: index/20201026.1.yml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+  - org: Talend Land Latam
+    url: https://www.talentland.talent-republic.tv
+subevents:
+  - title: Introduction to Bitcoin and Blockchain
+    start_utc: 2020-11-09 22:00
+    announcements:
+      - https://twitter.com/Decred_ES/status/1325908617770512385
+    presenters: Manuel Flores
+    media:
+      - https://www.youtube.com/watch?v=epZY2put3zQ
```
Comment (adcadde) [inline index/20201026.1.yml:25]: https://youtu.be/epZY2put3zQ

File: index/20201026.1.yml [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+  - title: Alternative Consensus Protocols
+    start_utc: 2020-11-10 22:00
+    announcements:
+      - https://twitter.com/Decred_ES/status/1326274399243218944
+    presenters: camilolwi
+    media:
+      - https://www.youtube.com/watch?v=rTjVKqKLI1w
```
Comment (adcadde) [inline index/20201026.1.yml:32]: https://youtu.be/rTjVKqKLI1w

File: index/20201026.1.yml [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  - title: Decentralized Organisations and Governance
+    start_utc: 2020-11-11
+    announcements:
+      - https://twitter.com/Decred_ES/status/1326633190442868737
+    presenters: adcade
+    media:
+      - https://www.youtube.com/watch?v=8GwzMhoEyO4
```
Comment (adcadde) [inline index/20201026.1.yml:39]: https://youtu.be/8GwzMhoEyO4

File: index/20201026.1.yml [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+  - title: Decred API and dcrdata workshop
+    start_utc: 2020-11-12
+    announcements:
+      - https://twitter.com/Decred_ES/status/1327000676505948166
+    presenters: pablito
+    media:
+      - https://www.youtube.com/watch?v=uuYf8Z28TcA
```
Comment (adcadde) [inline index/20201026.1.yml:46]: https://youtu.be/uuYf8Z28TcA

File: index/20201026.1.yml [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+  - title: How to become a Decred Contractor?
+    start_utc: 2020-11-13
+    announcements:
+      - https://twitter.com/Decred_ES/status/1327347110455480321
+    presenters: elian
+    media:
+      - https://www.youtube.com/watch?v=kqbejo0sA5c
```
Comment (adcadde) [inline index/20201026.1.yml:53]: https://youtu.be/kqbejo0sA5c

File: index/20201026.1.yml [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+  - 15 teams organized for the competition
+  - 6 teams presented projects in the final round
+description: The goal of the challenge is to share tools and practical abilities about blockchain technology in order for participants to develop an original project using Decred public blockchain data. First phase is one week of workshops about Bitcoin and blockchain, alternative consensus protocols, decentralised organisations and governance, dcrdata API and blockchain data, and Q&A party. Second phase is original project development with mentoring and support. At the end participants must release their projects with ISC license and present them in 3-minute video pitches.
+media:
+  - url: https://twitter.com/Decred_ES/status/1325817130978340868
+    note: Decred Talent Land Blockchain Challenge begins Nov-9
+  - url: https://www.youtube.com/watch?v=kBjgLcDZW8k
+    title_es: Lanzamiento del Reto Blockchain Learning Challenge
+    note: quick invitation and how to participate
+  - url: https://www.youtube.com/watch?v=czqQTw4VNBM
```
Comment (adcadde) [inline index/20201026.1.yml:77]: https://youtu.be/czqQTw4VNBM

**Review by adcadde — CHANGES_REQUESTED** [review]
Small changes on the youtube links and a few notes.

File: index/20201026.1.yml (hunk 4 above)
Comment (xaur) [reply index/20201026.1.yml:74]: Note that I am deliberately using full "canonical" YouTube links everywhere as a rule. This is esp. relevant for links that people end up posting on Reddit: submissions with exact same link are linked together and are shown on the Duplicates or "Other Discussions" tab. When links differ, even slightly, they are not linked.

File: index/20201026.1.yml (hunk 5 above)
Comment (xaur) [reply index/20201026.1.yml:25]: Same as above.

File: index/20201026.1.yml (hunk 6 above)
Comment (xaur) [reply index/20201026.1.yml:32]: Same as above.

File: index/20201026.1.yml (hunk 7 above)
Comment (xaur) [reply index/20201026.1.yml:39]: Same as above.

File: index/20201026.1.yml (hunk 8 above)
Comment (xaur) [reply index/20201026.1.yml:46]: Same as above.

File: index/20201026.1.yml (hunk 9 above)
Comment (xaur) [reply index/20201026.1.yml:53]: Same as above.

File: index/20201026.1.yml (hunk 10 above)
Comment (xaur) [reply index/20201026.1.yml:77]: Same as above.

**Comment by xaur** [comment]
Thanks for the review (and the challenge, ofc) @adcadde! This item is a big one but we're almost done. One last question remaining: was "final projects and winners" [video](https://www.youtube.com/watch?v=CQTitBVUMMY) livestreamed or was something people "connected to", or was it just posted on your YT channel? I am thinking whether it should be placed in "subevents" or "media".

**Comment by adcadde** [comment]
The video was posted on YouTube, therefore, better placed at "media". Thanks, bee!

**Comment by xaur** [comment]
Ok all good now, merged.

This was an epic index entry that really stress-tested my system and helped to shape the [spec](https://github.com/decredcommunity/events/blob/master/docs/index-spec.md) by formalizing the concept of "subevents".

Final link:

- https://decredcommunity.github.io/events/index/20201026.1

Don't hesitate to jump in #85 and share your experience/ideas to improve the index!

**Pull request closed without merging**